File: C/1045.c
#include <stdio.h>

int main(void) {
  double A, B, C;
  double caixa;
  scanf("%lf %lf %lf", &A, &B, &C);

  if(A < B){
    caixa = A;
    A = B;
    B = caixa;
  }  
  
  if(A < C){
    caixa = A;
    A = C;
    C = caixa;
  }

  if(B < C){
    caixa = B;
    B = C;
    C = caixa;
  }
  
  if(A >= (B + C)){
    printf("NAO FORMA TRIANGULO\n");
  }else{
    if( (A*A)==((B*B)+(C*C)) ){
      printf("TRIANGULO RETANGULO\n");
    }else if( (A*A)>((B*B)+(C*C)) ){
      printf("TRIANGULO OBTUSANGULO\n");
    }else if( (A*A)<((B*B)+(C*C)) ){
      printf("TRIANGULO ACUTANGULO\n");
    }
  } 
  
  if(A == B && B == C){
    printf("TRIANGULO EQUILATERO\n");
  }else if(A == B || B == C){
    printf("TRIANGULO ISOSCELES\n");
  }

  return 0;
}

File: Python/1045.py
a, b, c = input().split()

a = float(a)
b = float(b)
c = float(c)

if a < b:
    caixa = a
    a = b
    b = caixa

if a < c:
    caixa = a
    a = c
    c = caixa

if b < c:
    caixa = b
    b = c
    c = caixa

if (a >= (b+c)):
    print('NAO FORMA TRIANGULO')
else:
    if (a*a) == (b*b + c*c):
        print('TRIANGULO RETANGULO')
    elif (a*a) > (b*b + c*c):
        print('TRIANGULO OBTUSANGULO')
    elif (a*a) < (b*b + c*c): 
        print('TRIANGULO ACUTANGULO')

if a == b and b == c:
    print('TRIANGULO EQUILATERO')
elif a == b or b == c:
    print('TRIANGULO ISOSCELES')